### PR TITLE
fix: correct container image naming

### PR DESCRIPTION
**Pull request by jenting**
**What type of PR is this?**

/kind bug
/kind content

**Any specific area of the project related to this PR?**

/area documentation

**What this PR does / why we need it**:

The container image `falco-driver-loader` and `falco` description should be _falco-driver-loader_ and _falco-driver-loader included_.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/download.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For more details, see the [Run within Docker section](/docs/running#docker).
...
-|[latest](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:latest` | The most recent version of `falco-script-loader` with the building toolchain |
+|[latest](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:latest` | The most recent version of `falco-driver-loader` with the building toolchain |
 |[*version*](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:<version>` | A specific version of `falco-driver-loader` such as `{{< latest >}}` with the building toolchain |
 |[latest](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:latest` | The most recent version with the `falco-script-loader` included |
 |[*version*](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:<version>` | A specific version of Falco such as `{{< latest >}}` with `falco-script-loader` included |
```
Comment (leogr) [inline content/en/docs/download.md:49]:
```suggestion
|[*version*](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:<version>` | A specific version of Falco such as `{{< latest >}}` with `falco-driver-loader` included |
```

Another one here!

File: content/jp/docs/download.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Falcoは、実行中のシステムコールに関する情報を取得するた
...
 |[latest](https://hub.docker.com/r/falcosecurity/falco-no-driver/tags)| `docker pull falcosecurity/falco-no-driver:latest` | 最新バージョン |
 |[*version*](https://hub.docker.com/r/falcosecurity/falco-no-driver/tags)| `docker pull falcosecurity/falco-no-driver:<version>` | `{{< latest >}}`などの特定のバージョンのFalco |
-|[latest](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:latest` | ビルドツールチェーンを備えた `falco-script-loader`の最新バージョン |
+|[latest](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:latest` | ビルドツールチェーンを備えた `falco-driver-loader`の最新バージョン |
 |[*version*](https://hub.docker.com/r/falcosecurity/falco-driver-loader/tags)| `docker pull falcosecurity/falco-driver-loader:<version>` | ビルドツールチェーンを備えた`{{< latest >}}`などの  `falco-driver-loader`の特定のバージョン |
 |[latest](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:latest` | `falco-script-loader`が含まれる最新バージョン |
 |[*version*](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:<version>` |  `{{< latest >}}`のような、`falco-script-loader`が含まれた特定バージョンのFalco |
```
Comment (leogr) [inline content/jp/docs/download.md:49]:
```suggestion
|[*version*](https://hub.docker.com/r/falcosecurity/falco/tags)| `docker pull falcosecurity/falco:<version>` |  `{{< latest >}}`のような、`falco-driver-loader`が含まれた特定バージョンのFalco |
```

**Review by leogr — COMMENTED** [review]
Good catch! Thank you

Just found another two instance

**Comment by jenting** [comment]
@leogr I've updated my PR, PTAL :bow: 

BTW, I found that I have permission to approve myself PR, is this misconfigured or intended :thinking: ?

**Review by leogr — APPROVED** [review]
Thank you!

**Pull request merged**